pkg/proxy/ipvs/proxier.go

```diff
@@ -1599,6 +1609,8 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 	}
 	proxier.deleteEndpointConnections(endpointUpdateResult.StaleEndpoints)
+	metrics.SyncProxyRulesNoEndpointsTotal.WithLabelValues("internal").Set(float64(proxier.localNoEndpoints))
```

Comment:
The counter here and below is for "localNoEndpoints" and "clusterNoEndpoints", but the applied label values are "internal" and "external". If we're grouping by all internal or external, as opposed to "Local" or "Cluster", then we should rename the fields to `noEndpointsInternal` and `noEndpointsExternal`, similar to iptables

Reply:
Done. I've also renamed the metric to reflect that.